src/olympia/users/migrations/0006_auto_20210823_1247.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.6 on 2021-08-23 12:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0005_auto_20210601_1421'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='userprofile',
+            index=models.Index(fields=['last_login_ip'], name='users_last_login_ip_2cfbbfbd'),
+        ),
+        migrations.AddIndex(
+            model_name='userrestrictionhistory',
+            index=models.Index(fields=['ip_address'], name='u_restrict_hist_ip_4376df32'),
```

Comment:
we have `LongNameIndex` to avoid these abbreviated names, fwiw.  (applies to the other cryptically named indexed below too)

Reply:
Ah, I was trying to remember how we handled that... That's much better, I'll use that.